### DISCUSSION TITLE
Change Rights from "Optional" to "Recommended" or "Mandatory"

### DISCUSSION THREAD

**Discussion by KellyStathis** [datacite/datacite-suggestions#38] [Metadata Schema Suggestions]
*Note: This discussion consolidates multiple related suggestions.*

### What is the problem that your suggestion solves?

- We strongly support re-use of data in the context of Open Science, so this requires info on the rights of the data
- Better ethics, Machine readability for AI/ML, increase applications licences/Local Contexts labels,...

### What solution might meet your needs?

* We would like to see "Rights" become Mandatory 
* Boost the Rights metadata to Recommended (currently optional)

### What alternatives have you tried or considered?

We impose it to our research institutions, but they have difficulty finding this info because it is optional in the DataCite schema

### Is there anything else you would like to share?

Making rights "mandatory" we substantially help datasets to be re-used, and this would help Open Science to advance!

*Suggestion was previously submitted by a user. For DataCite staff reference: https://docs.google.com/spreadsheets/d/1ItM7bYocla83TCYmHI9oZ9Kgt7X0Lznfv6Zls7Gmi88/edit?resourcekey=&gid=1473745305#gid=1473745305*

### What group(s) would benefit from your suggestion?

- [ ] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by rebeccaringuette** [comment]
Completely agree that Rights should be a mandatory field. Items not indicating a license have a legal implication that no use is allowed, which causes misunderstandings.
